src/KeyChain.m

```diff
@@ -19,8 +19,6 @@
 //
 
 #import "KeyChain.h"
-#import <Security/SecKeychain.h>
```

Comment:
Not sure, it seems to compile just fine. Probably doesn't hurt to at least link the framework.